src/Analyzers/VisualBasic/Analyzers/VisualBasicAnalyzersResources.resx

```diff
@@ -129,4 +129,10 @@
   <data name="If_statement_can_be_simplified" xml:space="preserve">
     <value>'If' statement can be simplified</value>
   </data>
+  <data name="Add_invocation_parentheses" xml:space="preserve">
+    <value>Add parentheses to method invocation</value>
+  </data>
+  <data name="Remove_property_parentheses" xml:space="preserve">
+    <value>Remove parentheses from property invocation</value>
+  </data>
```

Comment:
@CyrusNajmabadi, I should have different message depending on whether the option is true or false. But the title is being sent in the ctor before I can determine the option value. What should I do here?

Reply:
code actions specify their titles.  so there can be four different values.

Reply:
@CyrusNajmabadi I meant in the analyzer not the code fix. The analyzer passes a title in the constructor before I can determine anything. Should I pass two Descriptors with the same id in the ctor? (One says `add parentheses to method invocation` and the other says `remove parentheses from method invocation`)

Reply:
the title you're passing is to make some default descriptor values that can be commonly used.  but you can make your own descriptors for hte other cases.

Reply:
@CyrusNajmabadi So it doesn't matter which title is passed to the ctor as long as I pass the specific title while reporting diagnostic, right?

Reply:
Yes.  the class is just optimized for teh common case of reporting one descriptor